### ISSUE TITLE
Potential BA.2*/BA.1*/BA.2* Recombinant with Double Breakpoints (244 Seqs in Israel, Germany, Canada, Ireland, Netherland, Sweden, Japan, India, UK and US as of 2022-06-10)

### ISSUE THREAD

**Issue by c19850727** [cov-lineages/pango-designation#590]
Although this potential recombinant sublineage doesn't meet the minimum criteria of 50 sequences, I thought it's still worth proposing, considering: 

- It first emerged back in January, and is still circulating (latest one sampled on April 12th and uploaded on April 25th);
- It's found in various US states and Canadian provinces.
- And it has unusual breakpoints respectively at ORF3a and ORF6/N protein.

### **Description**
**Recombinant between**: BA.2* & BA.1* & BA.2*
**Earliest sequence**: 2022/1/29 (Canada-AB)
**Most recent sequence**: 2022/4/12 (US-NY)
**Countries circulating**: Canada (AB, ON, SK, BC), US (NY, AL, HI, CA, MA)
**Likely breakpoint 1**: 25813-26088 (ORF3a) 
**Likely breakpoint 2**: 27385-29509 (ORF6/N protein). 

**Conserved Nuc mutations** (those in red frames are likely from BA.1*):
![image](https://user-images.githubusercontent.com/86835836/165657379-feb62f1d-f499-439f-9540-bcb6de1394c7.png)
**GISAID query**: NSP4_A146V, NSP1_S135R, Spike_A27S, N_S413R, NSP3_T24I, -NS3_T223I, -NS6_D61, M_D3G
**Covspectrum query**: 25731T, 26028T, 25810T, 29510C, 8991T, 670G, 26060C
### Evidence
**Usher tree**:
![image](https://user-images.githubusercontent.com/86835836/165658659-29b0dfb1-8e8a-4ff9-aa2f-95ac4e82aa5c.png)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_a814_9eac00.json?branchLabel=aa%20mutations&c=gt-nuc_3270,8991&label=nuc%20mutations:T26060C,A29510C

### Genomes:
EPI_ISL_9877333, EPI_ISL_11167367-11167368, EPI_ISL_11167399-11167401, EPI_ISL_11492717, EPI_ISL_11492879, EPI_ISL_11622851, EPI_ISL_11622857, EPI_ISL_11623289, EPI_ISL_11775999, EPI_ISL_11996156, EPI_ISL_12190831, EPI_ISL_12190878-12190879, EPI_ISL_12190882, EPI_ISL_12190906, EPI_ISL_12190918, EPI_ISL_12190927-12190928, EPI_ISL_12190953, EPI_ISL_12190956, EPI_ISL_12190975, EPI_ISL_12190998, EPI_ISL_12225720, EPI_ISL_12239090, EPI_ISL_12255633, EPI_ISL_12266581, EPI_ISL_12266638, EPI_ISL_12383373, EPI_ISL_12423243, EPI_ISL_12423309, EPI_ISL_12423316, EPI_ISL_12423478, EPI_ISL_12423653, EPI_ISL_12423679, EPI_ISL_12423757, EPI_ISL_12423774, EPI_ISL_12423901, EPI_ISL_12423962, EPI_ISL_12424004, EPI_ISL_12424309, EPI_ISL_12427093, EPI_ISL_12427913, EPI_ISL_12486748, EPI_ISL_12599558, EPI_ISL_12599568, EPI_ISL_12599572, EPI_ISL_12599617, EPI_ISL_12599844, EPI_ISL_12599846, EPI_ISL_12599858, EPI_ISL_12599892, EPI_ISL_12599926, EPI_ISL_12599994, EPI_ISL_12600075, EPI_ISL_12600413, EPI_ISL_12600426, EPI_ISL_12600430, EPI_ISL_12600468, EPI_ISL_12600671, EPI_ISL_12641010, EPI_ISL_12660743, EPI_ISL_12663960, EPI_ISL_12663962, EPI_ISL_12664055, EPI_ISL_12664092, EPI_ISL_12682961, EPI_ISL_12683057

**Comment by ktmeaton** [comment]
You have excellent timing, I was just drafting an issue for this same recombinant! I've added some output from [sc2rf](https://github.com/lenaschimmel/sc2rf) to look at the breakpoints.

```bash
python3 sc2rf.py nextclade.aligned.fasta --clades BA.1 BA.2 --ansi --unique 1
```

- I've excluded shared substitutions between both parents to make the breakpoints clearer.
- In this cluster (Issue#590) I've noticed 2 profiles. One has a "C" at 29510, and one has the reference allele "A".
- This differs from Issue#482 at position 26060 and thus has a distinct breakpoint.
- I've also added XQ, as an example of a designated recombinant with (similarly) few diagnostic positions for BA.1

![image](https://user-images.githubusercontent.com/14981272/165776482-0cb41778-514e-442d-af79-0f07b2dad1b2.png)

Mentioned issues:
- #482
- #468

**Comment by corneliusroemer** [comment]
How strong is the evidence for a double breakpoint? Does that rely on the single mutation 29510C? Or is there more evidence?

**Comment by ktmeaton** [comment]
From my analyses, I've only been reporting 1 breakpoint (24504:26059). I didn't see the S2M deletion in the AB samples, but I would defer to @c19850727 on this, as I'm inexperienced with indel analysis.

Here's some expanded sc2rf output with the `--enable-deletions` flag:

![image](https://user-images.githubusercontent.com/14981272/165840753-33819df4-cbe8-43a1-af73-32632bb5426e.png)

**Comment by FedeGueli** [comment]
@corneliusroemer there is N:S413R.

![Screenshot_20220428-230026_Chrome](https://user-images.githubusercontent.com/87669813/165844931-4bac454a-689a-4a15-a268-6bf98ff2b799.jpg)

**Comment by ktmeaton** [comment]
Time for some more verbose output (and a fresh clone of sc2rf for reproducibility). Possibly the Canadian samples (+1 USA AL) are a distinct recombinant clade? 

```bash
python3 sc2rf.py international_can.aligned.fasta --ansi --unique 1 --enable-deletions --breakpoints 0-100 --show-private-mutations
```

- The Vietnam sample has many private mutations in ORF1a and 1b, and includes a scattering of BA.1 alleles (ex. 5386G).
- The basal USA sample appears to have an earlier breakpoint, within Spike.
- The basal England samples doesn't have a confident breakpoint ("0 BP") due to high ambiguity. But also it has the BA.2 SM2 deletion beginning at 29734.

![image](https://user-images.githubusercontent.com/14981272/165856568-bd45f820-0650-4162-8e09-26a554c5ccfc.png)

Note: That tree on the left is hand-drawn to mentally connect the sc2rf output to the UShER tree.

**Comment by c19850727** [comment]
Thanks @ktmeaton and now I also tend to believe that those Canadian samples plus the US-AL one might be distinct.

Also the S2M deletion does look a bit tricky... (@corneliusroemer )

![image](https://user-images.githubusercontent.com/86835836/165860648-a24d689e-ba12-471b-8ec0-628d1a34039f.png)

**Comment by c19850727** [comment]
45 sequences as of 2022-05-01. The newest batch is uploaded from Canada-AB, Canada-BC, Canada-SK and US-HI.

**Comment by ktmeaton** [comment]
51 sequences as of 2022-05-02.

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_2ebc_a1d0.json?branchLabel=aa%20mutations&c=country&label=nuc%20mutations:C25731T,C25810T,C26028T

![image](https://user-images.githubusercontent.com/14981272/166291212-b1daa10e-6f5d-42e9-ad6c-bda03077da6a.png)

**Comment by ktmeaton** [comment]
I think this is going to be the cov-spectrum query for monitoring:

C25731T, C26028T, C25810T

https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?nucMutations=C25731T%2CC26028T%2CC25810T&

**Comment by ktmeaton** [comment]
I think I've figured out why I was missing the second breakpoint (and the S2M deletion) in sc2rf! With some minor code tweaks, I've caught up to your results (@c19850727).

- The Vietnam sample has A29514C but not the S2M deletion (interesting).
- The basal USA sample has a distinct and earlier breakpoint (23203:24129)
- The Candian clade (+3 USA) has one (24504:26059) or two breakpoints (27384:29510). 
   - The AB sequences are truncated after 29733 (no coverage).
   - The ON sequences have the deletion, and then resume coverage at 29760.
   - I'm not sure which is the simplest explanation (1 or 2 breakpoints?)

![image](https://user-images.githubusercontent.com/14981272/167203316-b63a36cb-f87d-4d3b-9a45-abbc698f1b2a.png)

**Comment by c19850727** [comment]
62 sequences as of 2022-05-09, with 14 sequences just uploaded from Canada-AB and 1 more from US-AZ.

**Comment by c19850727** [comment]
73 sequences as of 2022-05-10.

**Comment by c19850727** [comment]
I updated the Cov-spectrum/GISAID query, and revised the genome list, to exclude an unrelated cluster of potential recombinant (as noticed by @ktmeaton, many thanks!)

**GISAID query**: NSP4_A146V, NSP1_S135R, Spike_A27S, N_S413R, NSP3_T24I, -NS3_T223I, -NS6_D61, M_D3G
**Covspectrum query**: 25731T, 26028T, 25810T, 29510C, 8991T, 670G, 26060C

There is now 70 sequences as of 2022-05-13, that fit the updated query, and cluster well in the Usher tree below:
![image](https://user-images.githubusercontent.com/86835836/168418953-27a65193-53fb-4146-9a8f-e46dfb4d9fc8.png)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_1142b_f36a40.json?branchLabel=aa%20mutations&c=gt-nuc_25731&label=nuc%20mutations:C8991T

### Genomes:
EPI_ISL_9877333, EPI_ISL_11167367-11167368, EPI_ISL_11167399-11167401, EPI_ISL_11492717, EPI_ISL_11492879, EPI_ISL_11622851, EPI_ISL_11622857, EPI_ISL_11623289, EPI_ISL_11775999, EPI_ISL_11996156, EPI_ISL_12190831, EPI_ISL_12190878-12190879, EPI_ISL_12190882, EPI_ISL_12190906, EPI_ISL_12190918, EPI_ISL_12190927-12190928, EPI_ISL_12190953, EPI_ISL_12190956, EPI_ISL_12190975, EPI_ISL_12190998, EPI_ISL_12225720, EPI_ISL_12239090, EPI_ISL_12255633, EPI_ISL_12266581, EPI_ISL_12266638, EPI_ISL_12383373, EPI_ISL_12423243, EPI_ISL_12423309, EPI_ISL_12423316, EPI_ISL_12423478, EPI_ISL_12423653, EPI_ISL_12423679, EPI_ISL_12423757, EPI_ISL_12423774, EPI_ISL_12423901, EPI_ISL_12423962, EPI_ISL_12424004, EPI_ISL_12424309, EPI_ISL_12427093, EPI_ISL_12427913, EPI_ISL_12486748, EPI_ISL_12599558, EPI_ISL_12599568, EPI_ISL_12599572, EPI_ISL_12599617, EPI_ISL_12599844, EPI_ISL_12599846, EPI_ISL_12599858, EPI_ISL_12599892, EPI_ISL_12599926, EPI_ISL_12599994, EPI_ISL_12600075, EPI_ISL_12600413, EPI_ISL_12600426, EPI_ISL_12600430, EPI_ISL_12600468, EPI_ISL_12600671, EPI_ISL_12641010, EPI_ISL_12660743, EPI_ISL_12663960, EPI_ISL_12663962, EPI_ISL_12664055, EPI_ISL_12664092, EPI_ISL_12682961, EPI_ISL_12683057

**Comment by c19850727** [comment]
74 sequences as of 2022-05-17. Newly found in Germany-NW and US-MD.

**Comment by c19850727** [comment]
81 sequences as of 2022-05-19. Newly found in US-MN and US-UT.

**Comment by c19850727** [comment]
110 sequences as of 2022-05-20. All new sequences are from Canada-AB.
(hi @ktmeaton do you by any chance know if this is the result of targeted sequencing?)

**Comment by c19850727** [comment]
120 sequences as of 2022-05-25. Newly found in Netherland and Ireland.

**Comment by c19850727** [comment]
125 sequences as of 2022-05-26. Newly found in UK-England and US-KS.

**Comment by ktmeaton** [comment]
> hi @ktmeaton do you by any chance know if this is the result of targeted sequencing?)

Not that I know of, sorry :(

> Just an update that we're investigating why certain Canadian BA.2 sequences are missing 3' coverage that is needed to confidently call the S2M deletion (and therefore the second breakpoint).

The absence of the second breakpoint appears to be an artifact due to a combination of primer schemes, sequencing platform, and mapping errors. Sequences affected are BA.2 and its descendants (BA.3, BA.4, BA.5).

1. **Freed / Illumina**: Complete 3' dropout, no evidence of deletion.
2. **Freed / Nanopore**: Artifactual 3' dropout due to mapping error at read end. The final 9 bases that cover up to the last freed primer are shifted inside the deletion range.
3. **ARTIC / Illumina or Nanopore**: No 3' dropout, reads are long enough to span the deletion with correct mapping.

Current solutions to this problem we're pursuing:
  - Primer re-design, to shift the final Freed primer further downstream of the S2M motif.
  - Mapping to a reference with the S2M deletion removed, to reduce mapping errors.

tldr; The use of the [Freed/Midnight primer scheme](https://www.protocols.io/view/sars-cov2-genome-sequencing-protocol-1200bp-amplic-rm7vz8q64vx1/v6) leads to artifactual 3' dropout in BA.2 descendant lineages, due to the deletion of the stem-loop 2 motif (S2M). This is exacerbated by short read sequencing (i.e. Illumina). There are most likely 2 breakpoints in this recombinant lineage (proposed590), which is reproduced with samples sequenced using the [ARTIC](https://www.protocols.io/view/covid-19-artic-v4-1-illumina-library-construction-j8nlk4b36g5r/v2) primer scheme.

![proposed590_context_alignment](https://user-images.githubusercontent.com/14981272/171054495-4fa86b36-3bd7-475b-ab62-dafd8bf469a2.png)

**Comment by c19850727** [comment]
Thanks so much @ktmeaton for your follow-up and analysis!
Meanwhile, 18 sequences popped out today, mainly from Canada-AB and US-WA, but also newly found in Sweden.
Now in total there are 165 sequences as of 2022-05-30.

**Comment by c19850727** [comment]
171 sequences as of 2022-06-01, and now newly found in India-KA and Japan ex-US.

**Comment by FedeGueli** [comment]
This is now showing some advantage over BA.2.12.1 in the united states:
https://cov-spectrum.org/explore/United%20States/AllSamples/Past3M/variants?pangoLineage=BA.2.12.1*&nucMutations1=25731T%2C26028T%2C25810T%2C29510C%2C8991T%2C670G%2C26060C&analysisMode=CompareToBaseline&

Did it pick up some more mutations while circulating? I am monitoring since the first proposal and it seems its fitness vs other sublineages is increasing (the opposite of what we expect : usually growth advantage settles down while growing from cluster to lineage

**Comment by ktmeaton** [comment]
Thank you for catching and correcting that @AngieHinrichs! Just to clarify, the topology of `22A` (BA.4) and `22B` (BA.5) falling within the diversity of `21L` (BA.2) is not true to their evolutionary origin because there is underlying mutation conflicts? Or am I misinterpreting the tree or your explanation? Thanks!

https://nextstrain.org/ncov/gisaid/reference?label=clade:21M%20%28Omicron%29&m=div
![image](https://user-images.githubusercontent.com/14981272/171708141-6364bdf6-4e70-4b62-9b4f-9dafe617bdb6.png)

**Comment by AngieHinrichs** [comment]
21L on that tree is much broader than BA.2.  It includes recombinants (the sequences that appear below the 22A label; England/PHEC-YYFJF3T/2022 says BA.2 in the pop-up but it's XW), BA.4, BA.5, and BA.2 (the yellow-orange and red sequences above 22B).  21L is more like "Omicron minus BA.1 and BA.3".

I think it actually does show the separation of BA.2, BA.4 and BA.5, if you define BA.2 as the sequences above 22B.  Some reversions are still required  to get that structure -- the branch leading to 22A and 22B has a reversion on 23040 (shared by BA.1, BA.2 and BA.3), and the branch to 22B has 5 more reversions (again, tree structures can't be perfect when recombination might have been involved).  22A and 22B are also missing C9866T which is in nearly all BA.2 sequences, although as @corneliusroemer has pointed out, lack of C9866T might actually be ancestral for BA.2.

**Comment by ktmeaton** [comment]
Hi @InfrPopGen, I think we've resolved the major outstanding issue in this proposal with the consensus that this is a BA.1/BA.2 recombinant with double breakpoints. Are there any other problems to resolve before an official designation?

**Comment by c19850727** [comment]
205 sequences as of 2022-06-03, with 29 sequences newly uploaded from Canada-AB yesterday.

**Comment by c19850727** [comment]
216 sequences as of 2022-06-07, and newly found in Israel and US-CT.

**Comment by corneliusroemer** [comment]
@chrisruis @InfrPopGen Is there a reason this hasn't been designated yet? It's way above 50 sequences now, seems to be clean, in multiple countries. Would be great if you could have a quick look at this one while you're in the process of designating. Thanks!

**Comment by c19850727** [comment]
244 sequences as of 2022-06-10.

**Comment by InfrPopGen** [comment]
Thanks for submitting (and apologies for the delay). We've added recombinant lineage XAC with 126 newly designated sequences, and 3 updated designations.
